qa-test-app/src/main/res/drawable/bg_custom_info_panel2.xml

```diff
@@ -0,0 +1,4 @@
+<shape xmlns:android="http://schemas.android.com/apk/res/android" android:shape="rectangle">
+  <corners android:topLeftRadius="20dp" android:topRightRadius="20dp" />
```

Comment:
Maybe we can have smaller radius values - 5dp. Right now when toggling the style switch there is literally no difference in the background except for the margins.